emails/tests/models_tests.py

```diff
@@ -104,31 +106,31 @@ def setUp(self):
         self.premium_user_profile.server_storage = True
         self.premium_user_profile.save()
 
-    def test_make_relay_address_assigns_to_user(self):
-        relay_address = RelayAddress.make_relay_address(self.user_profile)
+    def test_create_assigns_to_user(self):
+        relay_address = RelayAddress.objects.create(user=self.user_profile.user)
         assert relay_address.user == self.user_profile.user
 
-    def test_make_relay_address_makes_different_addresses(self):
+    def x_test_create_makes_different_addresses(self):
```

Comment:
```suggestion
    def test_create_makes_different_addresses(self):
```
This test should pass since this is a valid use case of creating a random RelayAddress. When I test this locally this works so I'm unsure why you were ignoring this test case?